src/libstd/io/stdio.rs

```diff
@@ -872,6 +872,92 @@ impl fmt::Debug for StderrLock<'_> {
     }
 }
 
+/// Locks the current handle and reads a line of input, returning a `String` containing
```

Comment:
I think it would be good to keep this line (for documentation) 80 characters like the rest of the code so far I have see in rust.